mysql-test/suite/innodb/t/percona_changed_page_bmp_requests_debug.test

```diff
@@ -116,23 +139,31 @@ eval PURGE CHANGED_PAGE_BITMAPS BEFORE $previous_lsn;
 --source include/assert.inc
 
 --echo
---echo # Listing .xdb files after the restart with tracking disabled.
---echo # The output must show the same and a new .xdb file: ib_modified_log_7_nnn.xdb and ib_modified_log_8_nnn.xdb.
---replace_regex /_[[:digit:]]+\.xdb$/_<nnn>.xdb/
---list_files $CUSTOM_INNODB_DATA_HOME_DIR *.xdb
+--echo # Checking .xdb files after the restart with tracking disabled.
+--echo # There must stay some .xdb files.
+--let $prev_last_bmp_file_name = $last_bmp_file_name
+--source include/percona_get_last_bmp_file_name_size.inc
+
+if (`SELECT $bmp_files_count = 0`) {
+    --die "There must stay some .xdb files after restart (tracking enabled)"
+}
```

Comment:
How do we verify that there is a new .xdb file? Should we track the count before the restart and after and check the difference is 1?

Reply:
We could do so but when testing it I noticed that this number of files differ for different runs - sometimes there are as expected two files but sometimes there may stay only one but of bigger size. So I decided just to check that at least one file is present at this point. Looks like this should be enough. Do you think it is important to have exactly two files here?

Reply:
Ok, fine to keep it as is